pkg/config/agent/config.go

```diff
@@ -307,3 +309,23 @@ type PolicyBypassRule struct {
 	// The destination port of the given protocol.
 	Port int `yaml:"port,omitempty"`
 }
+
+type SecondaryNetworkConfig struct {
+	// Enable Antrea's native secondary network configuration.
+	Enable bool `yaml:"enable,omitempty"`
+	// Secondary network specific OVS configuration.
+	OVSConfig SecondaryNetworkOVS `yaml:"ovs,omitempty"`
+	// TunnelType to be used for node to node transport, which is part of the same virtual network.
+	TunnelType string `yaml:"tunnelType,omitempty"`
+}
+
+type SecondaryNetworkOVS struct {
```

Comment:
Probably name it `SecondaryNetworkOVSConfig ` to be clear?

Reply:
yeah. SecondaryNetworkOVS is what we had earlier. But, renamed it based on the other PR review comments (which got accidently closed when Adwait was resolving the conflicts).
We will rename it to SecondaryNetworkOVSConfig

Reply:
@adwaitni took care of the the above changes. @jianjuns @antoninbas please check and merge if there are no new comments.

Reply:
looking at this, the naming still seems wrong to me. I would expect `SecondaryNetworkConfig` and `SecondaryNetworkOVSConfig`.

Reply:
I'm kind of confused. we have 2 structs as below,

// This is the one we use it in AgentConfig - renamed it as SecondaryNetworkOVSConfig to reflect that its related to secondary network's OVS configuration (to diff it from SRIOV config).
type SecondaryNetworkOVSConfig struct {
        // Enable Antrea's native secondary network configuration.
        Enable bool `yaml:"enable,omitempty"`
        // Secondary network specific OVS configuration.
        OVSConfig SecondaryNetworkOVS `yaml:"ovs,omitempty"`
        // TunnelType to be used for node to node transport, which is part of the same virtual network.
        TunnelType string `yaml:"tunnelType,omitempty"`
}

// This is for secondary network specific OVS params. 
type SecondaryNetworkOVS struct {
        // OVS integration bridge name.
        OVSIntegrationBridgeName string `yaml:"ovsIntegrationBridgeName,omitempty"`
        // OVS transport bridge name.
        OVSTransportBridgeName string `yaml:"ovsTransportBridgeName,omitempty"`
        // OVS Datapath type to use for the OpenVSwitch bridge created by Antrea.
        OVSDatapathType string `yaml:"ovsDatapathType,omitempty"`
        // OVS patch port which connects the integration and transport bridge.
        OVSPatchPort string `yaml:"ovsPatchPort,omitempty"`
}

Reply:
I think we should name your first one as: `SecondaryNetworkConfig` and the second one as: `SecondaryNetworkOVSConfig`.